### PR TITLE
Update strategy for ODF upgrade testing

### PR DESCRIPTION
**Pull request by fbalak**

### CODE REVIEW

**Comment by fbalak** [comment]
Verified test_update_strategy_config_change: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/44102/

**Comment by fbalak** [comment]
Verified upgrade: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/44104

**Review by petr-balogh — DISMISSED** [review]

File: ocs_ci/ocs/ocs_upgrade.py [hunk 1]
```diff
@@ -858,3 +910,44 @@ def ocs_odf_upgrade_ui():
...
     pagenav_obj.odf_storagesystems_ui()
+
+
+def set_update_strategy(rbd_max_unavailable=None, cephfs_max_unavailable=None):
+    """
+    Update rook-ceph-operator-config configmap with parameters:
+    CSI_RBD_PLUGIN_UPDATE_STRATEGY_MAX_UNAVAILABLE and CSI_CEPHFS_PLUGIN_UPDATE_STRATEGY_MAX_UNAVAILABLE.
+    If values are not provided as parameters of this function then values are taken
+    from ocs-ci config. If the values are not set in ocs-ci config or function
+    parameters then they are updated.
```
Comment (ebondare) [inline ocs_ci/ocs/ocs_upgrade.py:921]: then they are _not_ updated?

Comment (fbalak) [reply ocs_ci/ocs/ocs_upgrade.py:921]: Right, good catch

**Review by petr-balogh — APPROVED** [review]

**Review by ebondare — APPROVED** [review]

**Comment by fbalak** [comment]
Verification run: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/44547/

**Pull request merged**